src/utilities/postData.js

```diff
@@ -0,0 +1,56 @@
+import { rerenderList } from './renderlist.js';
+
+export const addNewReservation = (id, userName, dateStart, dateEnd, url) => {
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      username: userName,
+      date_start: dateStart,
+      date_end: dateEnd,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((res) => res)
+    .then(() => rerenderList(id, `${url}?item_id=`))
+    .catch((err) => console.log('Network Error: ', err));
+};
+
+export default addNewReservation;
```

Comment:
- [ ] for javascript best practices, kindly avoid exporting the same function twice, `export const addNewReservation` & `export default addNewReservation;` , i would suggest removing the `export default addNewReservation;` since your current importing relying on  `export const addNewReservation` , kindly fix it 🙏